src/Donations/Micros/PlanetCashSelector.tsx

```diff
@@ -21,6 +22,7 @@ const PlanetCashSelector: FC = (props) => {
     paymentRequest,
   } = useContext(QueryParamContext);
   const router = useRouter();
+  const [planetCashCounrty, setplanetCashCountry] = useState();
```

Comment:
1. There's a typo in declaring `planetCashCountry`. Make sure to update elsewhere while correcting this.
2. Defining type while declaring a state variable would be useful. Currently I see a TS error while calling `setPlanetCashCountry`.
3. `planetCashCountry` and `country` always represent the same value. So perhaps we don't need a state variable here? Or a variable called `countryName` instead?